### ISSUE TITLE
Frameskipping for low-end devices

### ISSUE THREAD

**Issue by gunir** [mpv-player/mpv#14715]
### Expected behavior of the wanted feature

Hi, I've tried to search mpv.io and I happened there's no such feature.

For low-end devices, playing 60fps videos making their computer lag and overheat, and system becomes slower, it would be great to able to skip frame to lower fps to make it watchable and less system overhead.

**Comment by gunir** [comment]
> But if the bottleneck is decoding, this won't help.

Yeah, this is the hard part, would be great if there's a way to skip decoding too, that's what I expected and I've tried vf=fps, not really helpful for heavy format like AV1

**Comment by kasper93** [comment]
> not really helpful for heavy format like AV1

If your hardware is not capable of decoding given format in realtime there is nothing you can do.

For low end devices you will get best results with `--vo=gpu-next --profile=fast --hwdec` or `--vo=dmabuf-wayland --hwdec`.
